example/android/app/src/main/res/values/custom_theme.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ custom_theme.xml
+  ~
+  ~   PSPDFKit
+  ~
+  ~   Copyright © 2018-2019 PSPDFKit GmbH. All rights reserved.
+  ~
+  ~   THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
+  ~   AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
+  ~   UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
+  ~   This notice may not be removed from this file.
+  -->
+
+<resources>
+    <style name="PSPDFCatalog.Theme.Custom" parent="PSPDFKit.Theme">
```

Comment:
While looking through this PR and the existing sources on `master` I noticed that we don't actually use the name "Catalog" for the Flutter example app. I assume this was from moving themes from the Android catalog over to this example and would suggest that we rename it to feel more homey. My suggestion: `PSPDFKit.Theme.Flutter` as simple extension of the original `PSPDFKit.Theme` that serves as parent. WDYT?

Reply:
Yes, Catalog is probably misleading.

What about `PSPDFKit.Theme.Example`? This would help the users understand that is just a custom theme for our Flutter example app.